### DISCUSSION TITLE
Yanking

### DISCUSSION THREAD

**Discussion by Praczet** [ggandor/leap.nvim#138] [Q&A] [answered]
Hi,

Firstly I really like this plugin and I love Lightspeed. So thank you for all this work.

In [Lightspeed](https://github.com/ggandor/lightspeed.nvim) there was possibility to yank a text `yz{ch1}{ch2}<space>`.

Is it possible to do that in **Leap**?

**Comment by ggandor** [comment, marked as answer]
Of course, see `:help leap-operator-pending-mode`, `:help leap-default-mappings`. (The default mappings have been changed to `s/S` from `z/Z`.)

**Reply by Praczet** [reply]
Thank you for the reply. Indeed when I uninstalled nvim-surround it works..
